### PR TITLE
Edited too long lines. Fixes #76

### LINKED ISSUE #76: Don't ignore E722 and E501 flake8 errors
These ignores were introduced in #75 due to a new version of flake8. Need to get rid of the ignores and properly update the code.

### PR DESCRIPTION
**Pull request by asankov**

### CODE REVIEW

File: tcms/testruns/views.py [hunk 1]
```diff
@@ -460,7 +460,8 @@ def ajax_search(request, template_name='run/common/json_runs.txt'):
 
         # Get associated statistics data
         run_ids = [run.pk for run in searched_runs]
-        qs = TestCaseRun.objects.filter(run__in=run_ids).values('run').annotate(cases_count=Count('case'))
+        qs = TestCaseRun.objects.filter(run__in=run_ids)\
+            .values('run').annotate(cases_count=Count('case'))
```
Comment (atodorov) [inline tcms/testruns/views.py]: Elsewhere we use a different style for queries spanning multiple lines. For example:
```
qs = TestCaseRun.objects.filter(
    run__in=run_ids
).values(
    'run'
).annotate(
    cases_count=Count('case')
)
```

Please update it

File: tcms/xmlrpc/api/build.py [hunk 2]
```diff
@@ -57,9 +57,11 @@ def create(request, values):
 
     Example:
     # Create build by product ID and set the build active.
-    >>> Build.create({'product': 234, 'name': 'tcms_testing', 'description': 'None', 'is_active': 1})
+    >>> Build.create({'product': 234, 'name': 'tcms_testing', 'description': 'None', \
+                      'is_active': 1})
```
Comment (atodorov) [inline tcms/xmlrpc/api/build.py]: `\` is not necessary here

File: tcms/xmlrpc/api/build.py [hunk 3]
```diff
@@ -57,9 +57,11 @@ def create(request, values):
...
     # Create build by product name and set the build to inactive.
-    >>> Build.create({'product': 'TCMS', 'name': 'tcms_testing 2', 'description': 'None', 'is_active': 0})
+    >>> Build.create({'product': 'TCMS', 'name': 'tcms_testing 2', 'description': 'None', \
+                      'is_active': 0})
```
Comment (atodorov) [inline tcms/xmlrpc/api/build.py]: `\` here as well

**Comment by atodorov** [comment]
@asankov you will have to rebase after my latest changes. in `tcms/xmlrpc/views.py` there are some lines which were removed!

**Comment by asankov** [comment]
I have applied your comments and also rebased onto the newest changes.

**Pull request merged**